telemetryproxy/src/builder/config.yaml

```diff
@@ -33,7 +33,7 @@ processors:
   - gomod: "github.com/open-telemetry/opentelemetry-collector-contrib/processor/transformprocessor v0.103.0"
 
 replaces:
-  - github.com/open-telemetry/opentelemetry-collector-contrib/extension/lumigoauthextension v0.103.0 => github.com/lumigo-io/lumigo-otel-collector-contrib/extension/lumigoauthextension main
+  - github.com/open-telemetry/opentelemetry-collector-contrib/extension/lumigoauthextension v0.103.0 => github.com/lumigo-io/lumigo-otel-collector-contrib/extension/lumigoauthextension 70facbfc76f45838afc0d103e9c8c022a67d82c6
```

Comment:
This effectively pinpoints the collector to the version before https://github.com/lumigo-io/lumigo-otel-collector-contrib/pull/32, otherwise we'll need to upgrade the entire collector dependencies here which makes it a much bigger task.